docs/articles/expensify-classic/billing-and-subscriptions/Free-Trial.md

```diff
@@ -1,5 +1,44 @@
+
+To access these extra free weeks, all you need to do is complete the tasks on your Home page marked with the "Free Week!" banner. Each task completed in this category will automatically add seven more days to your trial. You can easily keep track of the remaining days of your Free Trial by checking the top right-hand corner of your Expensify Home page.
+
+# How to make the most of your Free Trial 
+*Complete all of the "Free Week!" tasks right away. These tasks are crucial for establishing your organization's Workspace, and finishing them will give you a clear idea of how much time you have left in your Free Trial. 
+
+*Every Free Trial has dedicated access to a Setup Specialist who can help you set up your account to your preferences. We highly recommend booking a call with your dedicated Setup Specialist as soon as you start your Free Trial. If you ever need assistance with a setup task, your tasks also include demo videos.
+
+*Invite a few employees to join Expensify as early as possible during the Free Trial. Bringing employees on board and having them submit expenses will allow you to fully experience how all of the features and functionalities of Expensify work together to save time. We provide excellent resources to help employees get started with Expensify.
+
+*Establish a connection between Expensify and your accounting system from the outset. By doing this early, you can start testing Expensify comprehensively from end to end.
+
+# FAQ
+## What happens when my Free Trial ends?
+If you’ve already added a billing card to Expensify, you will automatically start your organization’s Expensify subscription after your Free Trial ends. At the beginning of the following month, we'll bill the card you have on file for your subscription, adjusting the charge to exclude the Free Trial period.
+If your Free Trial concludes without a billing card on file, you will see a notification on your Home page saying, 'Your Free Trial has expired.' 
+If you still have outstanding 'Free Week!' tasks, completing them will extend your Free Trial by additional days. 
+If you continue without adding a billing card, you will be granted a five-day grace period after the following billing cycle before all Group Workspace functionality is disabled. To continue using Expensify's Group Workspace features, you will need to input your billing card information and initiate a subscription.
+
+## How can I downgrade my account after my Free Trial ends?
+If you’d like to downgrade to an individual account after your Free Trial has ended, you will need to delete any Group Workspace that you have created. This action will remove the Workspaces, subscription, and any amount owed. You can do this in one of two ways from the Expensify web app:
+-Select the “Downgrade” option on the billing card task on your Home page.
```

Comment:
It seems that this section isn't rendering properly as bullet points on new lines.
![image](https://github.com/Expensify/App/assets/26260477/6cacedfd-b847-4340-b445-68567ba63c24)

 The solution is to add a space after the dash, like this `- Select....`. Also please remove the space before the end of the bold section. For example `- Go to **Settings > Workspaces > [Workspace name]**`. After that it renders well.

![image](https://github.com/Expensify/App/assets/26260477/ed529765-f859-445b-b769-eadcd8d69d4a)

Reply:
Fixed!